packages/core/src/hooks/show/useShow.ts

```diff
@@ -22,15 +22,16 @@ import { pickResource } from "@definitions/helpers/pick-resource";
 import { useResource } from "../resource/useResource";
 import { pickNotDeprecated } from "@definitions/helpers";
 
-export type useShowReturnType<TData extends BaseRecord = BaseRecord> = {
-    queryResult: QueryObserverResult<GetOneResponse<TData>>;
+export type useShowReturnType<TQueryFnData extends BaseRecord = BaseRecord> = {
+    queryResult: QueryObserverResult<GetOneResponse<TQueryFnData>>;
```

Comment:
The usage below this file is correct (sending `TData` instead of `TQueryFnData`) but in this `useShowReturnType` type, naming of the generic is not quite right and hard to understand within the context of this file. (Might lead to confusion later in the future)

We can rename it to `TData` which is a better name and consistent with the `useShowProps`.

```suggestion
export type useShowReturnType<TData extends BaseRecord = BaseRecord> = {
    queryResult: QueryObserverResult<GetOneResponse<TData>>;
```

Reply:
@omeraplak @salihozdemir